internal/functions/Get-XpDirTreeRestoreFile.ps1

```diff
@@ -49,10 +49,10 @@ function Get-XpDirTreeRestoreFile {
     }
 
     if (!(Test-DbaPath -SqlInstance $server -path $path)) {
-        Stop-Function -Message "SqlInstance $SqlInstance cannot access $path" -EnableException $true
+        Stop-Function -Message "SqlInstance $SqlInstance cannot access $path" -EnableException $false
     }
     if (!(Test-DbaPath -SqlInstance $server -path $Path)) {
-        Stop-Function -Message "SqlInstance $SqlInstance cannot access $Path" -EnableException $true
+        Stop-Function -Message "SqlInstance $SqlInstance cannot access $Path" -EnableException $false
```

Comment:
I believe this syntax was previously problematic, switches should be `:$false` but in general, if it is `-EnableException:$false`, you wouldn't need to add the `-EnableException:$false` because it'd be false by default.